packages/hdwallet-core/src/cosmos.ts

```diff
@@ -56,6 +56,7 @@ export interface CosmosTx {
 
 export interface CosmosSignTx {
   addressNList: BIP32Path;
+  fee?: number;
```

Comment:
I like listing optionals last. not a big deal

Reply:
coo :+1: fixed